Nexus.Sdk.Token/Responses/TokenOperationResponse.cs

```diff
@@ -64,5 +66,11 @@ public TokenOperationResponse(string code, string hash, OperationAccountResponse
 
         [JsonPropertyName("paymentReference")]
         public string PaymentReference { get; set; }
+
+        [JsonPropertyName("fiatAmount")]
+        public decimal FiatAmount { get; private set; }
+
+        [JsonPropertyName("netFiatAmount")]
+        public decimal NetFiatAmount { get; private set; }
```

Comment:
Make it optional

Reply:
[Make fiat values optional in TokenOperationResponse](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/52/commits/fd7c22a7145780ca91153631128844dcddb72cc1)